scopes/lanes/merge-lanes/merge-abort.cmd.ts

```diff
@@ -10,11 +10,11 @@ export type MergeAbortOpts = {
 export class MergeAbortLaneCmd implements Command {
   name = 'merge-abort';
   description = `abort the recent lane-merge. revert the lane object and checkout accordingly`;
-  extendedDescription = `restore the lane-object same as it was before the last "bit lane merge" command.
-also, checkout the workspace components according to the restored lane`;
+  extendedDescription = `restore the lane-object to its state before the last "bit lane merge" command.
+also, checkout the workspace components according to the restored lane state`;
   alias = '';
   options = [
-    ['', 'verbose', 'show details of components that were not merged legitimately'],
+    ['', 'verbose', 'show details of components that were not merged successfully'],
```

Comment:
so here when components are not merged it can be because they failed or because there was no need to merge (e.g. not changed). This "verbose" is for the later.

Reply:
then i think it's better to switch that around - "legitimately not merged" - as otherwise sounds like they weren't merged in a legitimate way
WDUT?

Reply:
or even better, "components that didnt need to be merged"?

Reply:
Yeah, both work :)